modules/005-external-module-manager/hooks/handle_sources.go

```diff
@@ -361,6 +361,10 @@ func copyLayerToFS(rootPath string, rc io.ReadCloser) error {
 			continue
 		}
 
+		if !strings.Contains(hdr.Name, "..") {
```

Comment:
do you mean `strings.Contains` ?

maybe it's better to do something like
```
if strings.HasPrefix(hdr.Name, "..") {
```

Reply:
And probably we should return an error here. Cause we wouldn't get valid result otherwise

Reply:
Error, yes, but where? Just printing it to logs also does not make it better.
Storing it in the status is much better, but this is out of the scope of this PR. Because of the same reason, I did not add a test -- there are no tests for archive parsing.